tests/testapp/tests/test_permission_classes.py

```diff
@@ -117,3 +126,128 @@ def test_organization_field_with_errored_parent(self):
         with self.assertRaises(AttributeError) as error:
             self.client.get(reverse('test_error_field_view'), **auth)
         self.assertIn('Organization not found', str(error.exception))
+
+    def test_view_permission_with_operator(self):
+        user = User.objects.create_user(
+            username='operator', password='tester', email='operator@test.com'
+        )
+        operator_group = Group.objects.filter(name='Operator')
+        user.groups.set(operator_group)
```

Comment:
Just FYI, I think there are functions for common actions. Like [_create_operator](https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/tests/utils.py#L53).
These should be inherited and used! :smile:

Reply:
Hey, @atb00ker Yes, I tried to use it but with this function, the user gets all the permissions without being in the `operator` group, and here we only needed `view.testapp_template` permission.